### PR TITLE
Combine deployment docs into one main doc

### PR DESCRIPTION
**Pull request by elissa-alarmani**
In efforts to keep up with our documenting, we are moving towards having one main markdown file detailing all the documentation for deploying the gnomad browser (with addition to the blogs and reads). Right now this is combining `NewDeployment.md` `UpdateDeployment.md`  and `NewDemoDeployment.md` into `Deployment.md`

### CODE REVIEW

**Review by rileyhgrant — COMMENTED** [review]
Since these are docs, I'm going to leave a comment that I'd like you to review my edits and see if you agree/disagree with them. Adding commits to this seemed an easier way of communicating, as opposed to leaving many individual comments on specific lines of an `.md` file.

Let me know your thoughts on the changes!

**Comment by elissa-alarmani** [comment]
I agree with all of your changes, especially the blue/green deployment explanation, I wasn't exactly sure how to explain that since it was not in the original docs so I appreciate how you went about it. I'll convert this draft into a PR

File: deploy/docs/Deployment.md [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+
+## Prepare data
+
+See [data-pipeline/README.md](../../data-pipeline/README.md) for information on running data preparation pipelines.
+
+# Deploy on GKE
+
+## Create Elasticsearch cluster
+
+The setup step installs the [Elastic Cloud on Kubernetes (ECK)](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-overview.html) operator.
```
Comment (mattsolo1) [inline deploy/docs/Deployment.md:42]: Which setup step?

**Review by mattsolo1 — CHANGES_REQUESTED** [review]

**Comment by elissa-alarmani** [comment]
Apologies for the previous commits where I removed the ES docs`ElasticsearchConnection.md` and `ElasticsearchIndexAliases.md`, I misunderstood the purpose of keeping those docs separate from the deployment docs

**Review by mattsolo1 — APPROVED** [review]

**Pull request merged**